controllers/kyma_controller_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/kyma-project/eventing-auth-manager/controllers"
 	"github.com/kyma-project/eventing-auth-manager/internal/skr"
 	kymav1beta1 "github.com/kyma-project/lifecycle-manager/api/v1beta1"
+	kymav1beta2 "github.com/kyma-project/lifecycle-manager/api/v1beta2"
```

Comment:
Why do we need both versions?

Reply:
because v1beta1 now references v1beta2
https://github.com/kyma-project/lifecycle-manager/blob/d3b6e268f498587dffb44a14df528f1bc60e8753/api/v1beta1/kyma_types.go#L78

Reply:
I highly disagree with the implementation on `lifecycle-manger`s side, btw.